polyfill.js

```diff
@@ -1,7 +1,7 @@
 if (!Object.has) {
   var hasOwnProperty = Object.prototype.hasOwnProperty
 
-  Object.defineProperty(Object, "has", {
+  Object.defineProperty(Object, "hasOwn", {
     value: function(object, property) {
       return hasOwnProperty.call(object, property)
```

Comment:
```suggestion
      return hasOwnProperty.call(Object(object), property)
```
to reflect the updated exception ordering.

Reply:
It will kill an error on `null` / `undefined` and it's worse than the order of errors, in this case required an additional check.

Reply:
@jamiebuilds ^^^

Reply:
👀